### DISCUSSION TITLE
Vocprez endpoints

### DISCUSSION THREAD

**Discussion by recalcitrantsupplant** [RDFLib/prez#94] [General]
what are the vocprez endpoints and what can they deliver?

/v/vocab
/v/vocab/{vocab_curie}
/v/vocab/{vocab_curie}/{concept_curie}

/v/collection
/v/collection/{collection_curie}
/v/collection/{collection_curie}/{concept_curie} (only shows *direct* parent)  

Should concepts be listed under a collection? this would be a page which gives only the information for a single concept @nicholascar like this (but /v/collection/...) https://data.idnau.org/#/v/collection/6b9ae6e9cecedab5db392954961cf61d/816776141cd00178870f26fbe52fa0fd

**Comment by edmondchuc** [comment]
See https://github.com/RDFLib/prez/pull/88#issuecomment-1515775534 on why `/v/collection/{collection_curie}/{concept_curie}` was what we ended up with in the end.
